files/tasks/rpm-test-deps.yaml

```diff
@@ -25,3 +24,10 @@
     name: pytest-cov
   when: ansible_distribution != 'Fedora'
   become: true
+
+# flexmock is not in EPEL 9, install it from PyPI
```

Comment:
Please add the link to the https://bugzilla.redhat.com/show_bug.cgi?id=2120251 into the comment so that we can later check the progress.